testing/src/main/graphql/betterbotz/aggregations/GetOrdersAllFunctions.graphql

```diff
@@ -0,0 +1,23 @@
+query GetOrdersAllFunctions($filter: OrdersFilterInput) {
+    Orders(filter: $filter) {
+        pageState
+        values {
+            id
+            prodId
+            prodName
+            customerName
+            address
+            description
+            price
+            sellPrice
+            _int_function(name: "max", args: ["value_int"])
```

Comment:
Sorry about the conflicts, but I'm about to merge #906 and these new tests should definitely use the new "lightweight" approach.

Reply:
converted to a new approach.